helm/blueapi/values.yaml

```diff
@@ -119,7 +119,3 @@ initContainer:
     repositories: []
     # - name: "dodal"
     #   remote_url: https://github.com/DiamondLightSource/dodal.git
-
-# Mount path for scratch area from host machine, setting
-# this effectively enables scratch area management
-scratchHostPath: "" # example: /usr/local/blueapi-software-scratch
```

Comment:
Now that you've removed this value and made all the conditionals `{{- if .Values.initContainer.scratch.root }}`, if someone tried to run up the Helm chart with the default values `helm install blueapi ghcr://blueapi`, it will fail unless `/blueapi-plugins/scratch` is a path on the filesystem their Kubernetes mounts. 

One option is to add an `enabled` to `initContainer` and use that as the conditional check- this allows the self-documenting default to remain while not becoming a requirement of the minimal checkout- I think the Helm chart should always be in a state that `helm install blueapi` without any arguments or values file should work.

Make sure to also modify the documentation
https://diamondlightsource.github.io/blueapi/main/how-to/edit-live.html#kubernetes